### DISCUSSION TITLE
Quanta experiencia eu preciso para fazer pequenos trabalhos Frond End?

### DISCUSSION THREAD

**Discussion by ViksPIM** [frontendbr/forum#2388] [Perguntas e Respostas]
Bem, eu sou um Estudante de Frond End, estou aprendendo HTML, CSS e JS, e eu soube que em muitas escolas, os alunos começam a trabalhar assim que aprendem o básico e pagam por si mesmo a própria aprendizagem, eu gostaria de saber como começar, quais conhecimentos seriam necessarios? Quando eu deveria começar so para ganhar uma grana? Com que proposta eu devia começar para apresentar aos meus clientes?

**Comment by felquis** [comment]
Não tenho uma boa resposta, mas eu diria isso:

Se você está fazendo um curso, termine-o e tenta fazer alguns projetos pequenos para mostrar para as pessoas que você está entendendo de frontend.

Envie um email de apresentação para empresas e recrutadores ou qualquer pessoa que vc acha que pode te conectar a alguém que possa te contratar como iniciante/senior, e continue estudando. Assim eventualmente você encontra alguma sorte. Boa sorte.

**Comment by meow-os** [comment]
Não trabalho ainda pois estou no processo de aprendizagem ainda, mas recomendaria você fazer como o felquis disse: Crie projetos para mostrar que é capacitado e tente freelancing ou serviços para pessoas locais, por exemplo, um comerciante que não tem um site
